matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -289,14 +290,20 @@ internal class LocalEchoEventFactory @Inject constructor(
     }
 
     private fun createAudioEvent(roomId: String, attachment: ContentAttachmentData): Event {
+        val isVoiceMessage = attachment.mimeType == "audio/ogg"
```

Comment:
You can add a constant in `org.matrix.android.sdk.api.util.MimeTypes`

Reply:
This is also not the proper way to tell if it's a voice message or not. Add a new field boolean in  `ContentAttachmentData` instead?

Reply:
Changed to `attachment.waveform != null`